timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -127,14 +204,165 @@ export default {
       return this.$store.state.meta
     },
   },
+  created() {
+    if (this.timeline) {
+      ApiClient.getSketchTimelineAnalysis(this.sketch.id, this.timeline.id)
+        .then(response => {
+          this.analyses = response.data.objects[0]
+        })
+        .catch(e => { })
+    }
+    // If no timeline was specified then loop over all of them
+    else {
+      this.sketch.timelines.forEach(timeline => {
+        ApiClient.getSketchTimelineAnalysis(this.sketch.id, timeline.id)
+          .then(response => {
+            this.analyses = this.analyses.concat(response.data.objects[0])
+          })
+          .catch(e => { })
+      })
+    }
+  },
+  mounted() {
+    this.loadSketchSigmaTags()
+    this.loadSketchTTP()
+  },
   methods: {
-    parseSigma: function(event) {
-      ApiClient.getSigmaByText(this.text)
+    problem_detector: function (search_query) {// eslint-disable-line
```

Comment:
```suggestion
    problemDetector: function (searchQuery) {// eslint-disable-line
```

Reply:
why are you asking eslint to not format this? Can we come up with a more descriptive name for this function? e.g. `extractWarningsFromRule`